easybuild/easyconfigs/b/BOLT-LMM/BOLT-LMM-2.3.4-foss-2020b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'BOLT-LMM'
+version = '2.3.4'
+
+homepage = 'https://storage.googleapis.com/broad-alkesgroup-public/BOLT-LMM/BOLT-LMM_manual.html'
+description = """The BOLT-LMM software package currently consists of two main algorithms, 
+the BOLT-LMM algorithm for mixed model association testing, and the BOLT-REML algorithm 
+for variance components analysis (i.e., partitioning of SNP-heritability and estimation 
+of genetic correlations)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = [
+    'https://storage.googleapis.com/broad-alkesgroup-public/%(name)s/downloads/',
+    'https://storage.googleapis.com/broad-alkesgroup-public/%(name)s/downloads/old/'
+]
+sources = ['%(name)s_v%(version)s.tar.gz']
+checksums = ['d7b587ea1a3cc950084c3cc8bfd9b8a75ca90899ee286261e9ea7a2c7f4c777a']
+patches = ['BOLT-LMM-fix-makefile.patch']
```

Comment:
@btlindert please add the missing checksum for the patch.

Reply:
@verdurin thanks for your comments. I've added the checksums and the patch explanation.